<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemWizardData</class>
 <widget class="QDialog" name="SystemWizardData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GEM-Selektor System</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="pageName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Step 1 - System configuration (thermodynamic data)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="textLabel12">
         <property name="text">
          <string>Modelling Project is a named set of GEMS database records for a specific modelling application, whose main is a generic definition of chemical thermodynamic systems. This includes lists of record keys for ICs, DCs, phases etc. kept in the project files. 
   The project configuration record contains links to GEMS database files where the input data is kept and the modelling results will be stored.These database files are located in an (automatically created) subdirectory having the same name as the modelling project. 
   A selection of internally calculated thermodynamic data, plus project-specific numerical settings and default values, can be set up in this wizard. Thermodynamic data currently used in CSDs are not saved and will be calculated again whenever necessary.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Full name of System</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="pName"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="pComment"/>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Allocation flags for using additional thermodynamic data in CSDs </string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="pPE">
            <property name="text">
             <string>electroneutrality constraint PE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="pPV">
            <property name="text">
             <string>volume constraint PV</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="pXr0h0">
            <property name="text">
             <string>Xr0h0_ parameters r0 and h0 of particles (pores)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="pPBeq">
            <property name="text">
             <string>Account for electrostatic sorption models (DDLM,CCM,TLM)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="pPXepsC">
            <property name="text">
             <string>XepsC_ diel.const &amp; el.conductivity of phase carriers</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="pPAalp">
            <property name="text">
             <string>Aalp_ specific surface areas for phases</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="pPParc">
            <property name="text">
             <string>Pparc_ partial pressures for DC</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="pPSigm">
            <property name="text">
             <string>Sigm_ specific surface energy for solid phase-aqueous(gas)interface</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="pPDelB">
            <property name="text">
             <string>delB_  uncertainties of bulk composition (vector b) elements</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="pPNfsp">
            <property name="text">
             <string>Nfsp_,SCMT_: fractions and SCM codes for surface types</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="pPXlam">
            <property name="text">
             <string>Xlam_ EDL discretness parameter for surface types, &gt;0, &lt;1</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="pPbPh">
            <property name="text">
             <string>xp_ via quantities of phases from EQSTATe pointed by rkey in SyPhEQ</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="pPLim">
            <property name="text">
             <string>lower and  upper restrictions for X_a (phases)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Step 2 - System configuration (hints)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Key of EQSTATe record to supply phases for b vector calculation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pKey"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>What will happen after you click &quot;Finish&quot;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="textLabel13">
            <property name="text">
             <string>(1) The configutarion of modeling project will become active, and later on will be saved in the project configuration file in project directory.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel13_2">
            <property name="text">
             <string>(2) The Project module window will remain open. In most cases, you should just close it and proceed working with the modeling project.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel13_3">
            <property name="text">
             <string>(3) Before closing the Project module window, you may wish to additionally change some default settings on pages Param_1 or Param_2. This will affect newly created records in the respective modules; old records will remain intact. Do not forget to close the Project window then.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBack">
       <property name="text">
        <string>&lt; &amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pNext">
       <property name="text">
        <string>&amp;Next &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pPE</tabstop>
  <tabstop>pPV</tabstop>
  <tabstop>pPBeq</tabstop>
  <tabstop>pPAalp</tabstop>
  <tabstop>pPSigm</tabstop>
  <tabstop>pPNfsp</tabstop>
  <tabstop>pPLim</tabstop>
  <tabstop>pXr0h0</tabstop>
  <tabstop>pPXepsC</tabstop>
  <tabstop>pPParc</tabstop>
  <tabstop>pHelp</tabstop>
  <tabstop>pBack</tabstop>
  <tabstop>pNext</tabstop>
  <tabstop>pCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pCancel</sender>
   <signal>clicked()</signal>
   <receiver>SystemWizardData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
